pkg/runtime-manager/server/cri/criserver.go

```diff
@@ -0,0 +1,137 @@
+/*
+
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cri
+
+import (
+	"context"
+	"net"
+	"time"
+
+	"google.golang.org/grpc"
+	runtimeapi "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+	"k8s.io/klog/v2"
+
+	"github.com/koordinator-sh/koordinator/pkg/runtime-manager/config"
+	"github.com/koordinator-sh/koordinator/pkg/runtime-manager/dispatcher"
+	resource_executor "github.com/koordinator-sh/koordinator/pkg/runtime-manager/resource-executor"
+	"github.com/koordinator-sh/koordinator/pkg/runtime-manager/server/utils"
+)
+
+const (
+	defaultTimeout = 5 * time.Second
+)
+
+type RuntimeManagerCriServer struct {
+	hookDispatcher *dispatcher.RuntimeHookDispatcher
+	runtimeClient  runtimeapi.RuntimeServiceClient
```

Comment:
seems we use it to communicate with hook server? change to hookClient, WDYT?

Reply:
> seems we use it to communicate with hook server? change to hookClient, WDYT?

runtimeClient is used  to talk with containerd not hookserver